docs/getting-started-on-the-raspberry-pi.md

```diff
@@ -0,0 +1,104 @@
+# Getting Started on the Raspberry Pi 3
+
+This guide assumes you are using the [Raspbian image]. It should work on other
+distributions as well, but some details may be different.
+
+We will compile the driver on the Raspberry Pi itself, of course you can also
+cross compile from your system. 
+
+## Download Driver
+
+Get the latest release from
+https://github.com/Sensirion/embedded-uart-sps/releases (sps30-uart-x.y.z.zip)
+
+```bash
+# on the Raspberry Pi
+$ wget https://github.com/Sensirion/embedded-uart-sps/releases/download/1.0.0/sps30-uart-1.0.0.zip
+$ unzip sps30-uart-1.0.0.zip
+```
+
+## Get Port of the Connected Sensor
+
+We need to now the port of the connected sensor. After connecting the sensor
+via USB run the following:
+```bash
+# on the Raspberry Pi
+$ dmesg | grep usb
```

Comment:
Isn't `ls /dev/ttyUSB*` simpler? I think there's no other USB tty by default on a raspi..

Reply:
Is it guaranteed that it shows up as `/dev/ttyUSB`? I have also seen stuff like `/dev/ttyACM0`.

Reply:
with both official cables yes, IIRC, ACM is used if the device also supports modem-like AT commands.

Reply:
The dmesg has the advantage that it is also a good way for debugging any issues with the USB device, so I'd rather keep it.

Reply:
I'd say we can keep this for now, and improve it later on if it is confusing to users